src/libs/TransactionUtils.js

```diff
@@ -175,9 +175,11 @@ function getCurrency(transaction) {
  */
 function getCreated(transaction) {
     const created = lodashGet(transaction, 'modifiedCreated', '') || lodashGet(transaction, 'created', '');
-    if (created) {
-        return format(new Date(created), CONST.DATE.FNS_FORMAT_STRING);
+    const createdDate = parseISO(created);
+    if (createdDate) {
```

Comment:
Use `isValid` from date-fns to better check if the date is valid. `createdDate` could be an 'Invalid Date'.
```suggestion
    if (isValid(createdDate)) {
```